src/components/Films/Films.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Select, Table, Col, Input, Typography } from 'antd';
+
+import { getFilmDataByGenre, getFilmDataByQuery } from '../Redux/Action';
+import { RootState } from '../../Reducers';
+import { columns } from '../Config/filmsTable.config';
+import { genres } from '../Config/filmsGenres.config';
+
+import styles from './Films.module.scss';
+import 'antd/dist/antd.css';
+
+export interface IMovieDataItems {
+  adult: boolean;
+  backdrop_path: string;
+  genre_ids: number[];
+  id: number;
+  original_language: string;
+  original_title: string;
+  overview: string;
+  popularity: number;
+  poster_path: string;
+  release_date: string;
+  title: string;
+  video: boolean;
+  vote_average: number;
+  vote_count: number;
+}
+
+const { Title } = Typography;
+
+export const Films = () => {
+  const data = useSelector((state: RootState) => state.film).data;
+  const [title, setTitle] = useState<string>('');
+  const [genre, setGenre] = useState<number>(16);
+
+  const { Option } = Select;
+
+  const dispatch = useDispatch();
+
+  const getFilmInfo = () => {
+    dispatch(getFilmDataByGenre(genre));
+  };
+
+  const getFilmBySearch = () => {
+    dispatch(getFilmDataByQuery(title));
+  };
+
+  useEffect(() => {
+    getFilmInfo();
+  }, [genre]);
+
+  useEffect(() => {
+    getFilmBySearch();
+  }, [title]);
+
+  const movieData: object[] = [];
```

Comment:
do you have the interface what is the type of this data ? maybe object has structured type